src/routes/modules/+page.svelte

```diff
@@ -35,10 +38,25 @@
 			});
 	});
 
+	function limitResults() {
+		let allRows = document.querySelectorAll('.module-row');
+		allRows.forEach(function (row) {
+			row.classList.remove('length-filtered-out');
+		});
+		console.log('showAll', showAll);
+		if (!showAll) {
+			let results = document.querySelectorAll('.module-row:not(.search-filtered-out)');
+			for (let i = resultLimit; i < results.length; i++) {
+				results[i].classList.add('length-filtered-out');
+			}
+		}
+	}
```

Comment:
Didn't you implement this into LayoutSearchFilter, why does it need to be done again here?

Reply:
Great question. the one in LayoutSearchFilter does not know the order which elements are laid out, which is fine if the sort order never changes, like on the auditlog page, but on pages like /modules, when the sort order changes, the layout filter will not hide the correct ones.